src/transaction.ts

```diff
@@ -287,9 +378,39 @@ const unsignedTxRLP = new RLP({
     ],
 })
 
-const txRLP = new RLP({
-    name: 'tx',
-    kind: [...(unsignedTxRLP.profile.kind as RLP.Profile[]), { name: 'signature', kind: new RLP.BufferKind() }],
+const legacyTxRLP = new RLP({
+    name: 'legacy tx',
+    kind: [...(unsignedLegacyTxRLP.profile.kind as RLP.Profile[]), { name: 'signature', kind: new RLP.BufferKind() }],
+})
+
+const unsignedDynamicFeeTxRLP = new RLP({
+    name: 'unsigned dynamic fee tx',
+    kind: [
+        { name: 'chainTag', kind: new RLP.NumericKind(1) },
+        { name: 'blockRef', kind: new RLP.CompactFixedBlobKind(8) },
+        { name: 'expiration', kind: new RLP.NumericKind(4) },
+        {
+            name: 'clauses', kind: {
+                item: [
+                    { name: 'to', kind: new RLP.NullableFixedBlobKind(20) },
+                    { name: 'value', kind: new RLP.NumericKind(32) },
+                    { name: 'data', kind: new RLP.BlobKind() },
+                ],
+            },
+        },
+        { name: 'maxPriorityFeePerGas', kind: new RLP.NumericKind(32) },
```

Comment:
Is `32` enough? Maybe `256` could be more appropriate? (for both maxPriorityFeePerGas and maxFeePerGas)

Reply:
`32` is the byte length here in the ` RLP.NumericKind` thus it's 256 bit, so enough for the fee fields